aiopinboard/bookmark.py

```diff
@@ -87,7 +87,7 @@ async def async_add_bookmark(  # pylint: disable=too-many-arguments
         if description:
             params["extended"] = description
         if tags:
-            params["tags"] = tags
+            params["tags"] = " ".join(tags)
```

Comment:
Do we need to do anything special if the user provides a tag that already has a space in it?

Reply:
I guess it could be a thing to do; [but the API doesn't allow spaces in tags](https://pinboard.in/api/#entity_tag) and if you were to type a space into the main UI on the site it would take it to be two tags. So if that were to be done then `aiopinboard` would need to invent its own conversion convention. Personally, as someone building a client with this, I'm happy for them to pass right through because I'm using space to separate the tags anyway, just as the main site does:

![Screenshot 2023-12-13 at 14 27 10](https://github.com/bachya/aiopinboard/assets/28237/bf2fb5d4-bc95-4135-9104-91f9d6371507)

Reply:
(thinking some more: if anything I'd say that `aiopinboard` should probably throw an error if asked to use a tag that contains a space or a comma)

Reply:
I'm aligned. Could you open a separate PR for that?

Reply:
Sure, happy to do that. Might be a day or so (and normally when I say that it ends up being the same day but still...).